### PR TITLE
Eng 13096 kafka importer junit

### PR DESCRIPTION
**Pull request by xinjiacs**
Add Junit test for kafka importer

### CODE REVIEW

File: template.classpath [hunk 1]
```diff
@@ -77,5 +77,9 @@
 	<classpathentry kind="lib" path="lib/scala-parser-combinators_2.11-1.0.2.jar"/>
 	<classpathentry kind="lib" path="lib/scala-xml_2.11-1.0.2.jar"/>
 	<classpathentry kind="lib" path="third_party/java/jars/javassist-3.20.0-GA.jar"/>
+        <!-- zkclient is used for test only -->
+	<classpathentry kind="lib" path="third_party/java/jars/zkclient-0.10.jar"/>
```
Comment (akhanzode) [inline template.classpath]: Do you want to add to README.thirdparty? Also is it Ok to add these licensewise?

File: tests/frontend/org/voltdb/regressionsuites/TestKafkaImportSuite.java [hunk 2]
```diff
@@ -0,0 +1,257 @@
...
+        while (sw.elapsed(TimeUnit.SECONDS) < (30)) {
+            VoltTable vt = client.callProcedure("@AdHoc", "Select * from tmap order by val;").getResults()[0];
+            System.out.println("Elapsed " + sw.elapsed(TimeUnit.SECONDS) + " seconds, Test table contents: " + vt);
+            if (10 == vt.getRowCount()) {
+                findImportedData = true;
+                break;
+            }
+            Thread.sleep(1000);
+        }
+        assertTrue("VoltDB Kafka importer does not import enough data", findImportedData);
```
Comment (akhanzode) [inline tests/frontend/org/voltdb/regressionsuites/TestKafkaImportSuite.java]: should be "did not import enough data"

File: tests/frontend/org/voltdb/regressionsuites/TestKafkaImportSuite.java [hunk 3]
```diff
@@ -0,0 +1,257 @@
...
+            expectedMessageList.add(message);
+            KeyedMessage<String, String> data =
+                    new KeyedMessage<String, String>(TEST_TOPIC, message);
+            producer.send(data);
+        }
+        Client client = getClientToHostId(0);
+
+        // check Kafka importer result
+        Stopwatch sw = Stopwatch.createStarted();
+        boolean findImportedData = false;
```
Comment (akhanzode) [inline tests/frontend/org/voltdb/regressionsuites/TestKafkaImportSuite.java]: foundImportData

File: tests/frontend/org/voltdb/regressionsuites/TestKafkaImportSuite.java [hunk 4]
```diff
@@ -0,0 +1,257 @@
...
+                findImportedData = true;
+                break;
+            }
+            Thread.sleep(1000);
+        }
+        assertTrue("VoltDB Kafka importer does not import enough data", findImportedData);
+        producer.close();
+    }
+
+    public void testProcedureConsumer() throws Exception {
```
Comment (akhanzode) [inline tests/frontend/org/voltdb/regressionsuites/TestKafkaImportSuite.java]: This test is not testing any volt functionality and can be removed.

Comment (xinjiacs) [reply tests/frontend/org/voltdb/regressionsuites/TestKafkaImportSuite.java]: I will add a comment to this junit and disable it maybe. This is an example code showing how to use producer and consumer from kafka cluster inside a junit.

File: template.classpath (hunk 1 above)
Comment (xinjiacs) [reply template.classpath]: It's just used for testing and also used as a binary file. Apache license should be fine with that.

**Pull request merged**